.github/dependabot.yaml

```diff
@@ -0,0 +1,39 @@
+version: 2
+updates:
```

Comment:
This file has been edited across multiple commits with instances of code being removed and then re-added. To streamline the review, the offending commits should be removed, either by rebasing or squashing them into a single, meaningful commit.